apps/docusaurus/docs/nodes/full-node/network-identity-fullnode.md

```diff
@@ -11,15 +11,21 @@ PFN identities are more ephemeral and can be regenerated on demand. As such, gen
 guide **should only be done for PFNs**, and not for validators or VFNs.
 :::
 
-Public fullnodes (PFNs) will automatically start up with a randomly generated (ephemeral) network identity. This works well for
-regular PFNs. However, you may want to generate and assign a static network identity to your PFN. This is useful when:
+## Ephemeral vs. Static Identities
+Public fullnodes (PFNs) will automatically start up with a randomly generated (ephemeral) network identity unless a static identity is provided. This works well for
+regular PFNs. However, there are cases where you may want to generate and assign a static network identity to your PFN.
+
+### Ephemeral Identity
+
+- Automatically generated on startup. The same ephemeral identity is used across restarts if the identity key file already exists.
+- Stored at `/opt/aptos/data/db/ephemeral_identity_key`.
+
+### Static Identity
```

Comment:
Please update the associated page in Nextra as well `apps/nextra/pages/en/blockchain/nodes/...`